arch/arc/Kconfig

```diff
@@ -78,6 +75,7 @@ config CPU_EM6
 config CPU_HS3X
 	bool
 	select CPU_ARCHS
+	select HAS_FLASH_LOAD_OFFSET
```

Comment:
How this hunk is related to atomics?

Reply:
I wonder why and when I add this too. removed.